tokio-macros/src/entry.rs

```diff
@@ -235,6 +277,9 @@ fn parse_knobs(
     if let Some(v) = config.worker_threads {
         rt = quote! { #rt.worker_threads(#v) };
     }
+    if let Some(v) = config.start_paused {
+        rt = quote! { #rt.start_paused(#v) };
+    }
```

Comment:
Since making the test deterministic requires calling it on the builder, I agree that it should be added to the macro, but we should document it on both `#[tokio::test]` and `tokio::time::pause`.

Reply:
Done